tests/syscalls/getpid_test.cc

```diff
@@ -19,17 +19,11 @@
 #include <stdlib.h>
 #include <unistd.h>
 
-#ifdef USE_RAW_SYSCALLS
-#define GETPID NACL_SYSCALL(getpid)
-#else
-#define GETPID getpid
-#endif
-
 bool TestGetPid() {
   bool test_status;
   const char *testname = "getpid_test";
-  pid_t pid_one = GETPID();
-  pid_t pid_two = GETPID();
```

Comment:
You can add a comment to show that we are ignoring this on purpose and you can also change the comment on line 9.

Reply:
Fixed.